cli/tre/commands/shared_services/shared_service.py

```diff
@@ -10,8 +10,17 @@
 from .operations import shared_service_operations
 
 
+def shared_service_id_completion(ctx, param, incomplete):
```

Comment:
nit: This is fine....it's just that you've been type-hinting all the other methods, so I wondered "why not this one?"

Reply:
I've pushed a commit with type annotations added to the completion methods